.github/workflows/benchmark.yml

```diff
@@ -4,7 +4,7 @@ on:
     branches:
       - main
 env:
-  DEFAULT_GO_VERSION: 1.16
+  DEFAULT_GO_VERSION: 1.17
```

Comment:
Can you please explain why `1.17`?
Why not `1.18`?

Reply:
I believe that is because the default Go version is the minimum supported version. By the way, we still support `1.16`, so the minimum version of Go to support should be `1.16`.

Reply:
I did this so we won't be surprised by the difference between race tests and linting at 1.17 vs 1.16. we can delay this until the removal of 1.16. I'll put a patch in the morning.